### PR TITLE
Updated HTTP method validation in Controller decorator

### PR DESCRIPTION
**Pull request by eatmoreapple**

### CODE REVIEW

File: nest/core/decorators/controller.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import http
```
Comment (ItayTheDar) [inline nest/core/decorators/controller.py:1]: You declare this import but never use it. could you remove this?

File: nest/core/decorators/controller.py [hunk 2]
```diff
@@ -26,52 +28,22 @@ def Controller(tag: str = None, prefix: str = None):
...
+                assert hasattr(method, "__path__") and method.__path__, f"Missing path for method {name}"
+
+                http_method = method.method
+                # Ensure that the method is a valid HTTP method
+                assert http_method in http_method_names, f"Invalid method {http_method}"
+                if prefix:
+                    method.__path__ = prefix + method.__path__
+                if not method.__path__.startswith("/"):
+                    method.__path__ = "/" + method.__path__
+                router.add_api_route(method.__path__, method, methods=[http_method], **method.__kwargs__)
```
Comment (ItayTheDar) [inline nest/core/decorators/controller.py:45]: please run black on this file to make format align with rest of the project

**Review by ItayTheDar — CHANGES_REQUESTED** [review]
Thank you so much for this contribution! this is a really nice work, that simplifies the controller decorator

File: nest/core/decorators/controller.py (hunk 2 above)
Comment (eatmoreapple) [reply nest/core/decorators/controller.py]: I hadn't used this before, but I find it extremely useful.

**Review by ItayTheDar — APPROVED** [review]

**Pull request merged**